tests/e2e/scenarios/upgrade-ab/run-test.sh

```diff
@@ -121,24 +121,35 @@ else
   "${KOPS_B}" edit cluster "${CLUSTER_NAME}" "--set=cluster.spec.kubernetesVersion=${K8S_VERSION_B}"
 fi
 
-"${KOPS_B}" update cluster
-"${KOPS_B}" update cluster --admin --yes
-# Verify no additional changes
-"${KOPS_B}" update cluster
+if [[ "${KOPS_VERSION_B}" =~ 1.3[123456789] ]]; then
```

Comment:
maybe we should invert this, using the old method when the version matches `1.2` or `1.30`. That way we avoid surprises when 1.40 is tested.